test_unstructured/documents/test_base.py

```diff
@@ -25,6 +37,12 @@ def test_get_narrative():
     document.print_narrative()
 
 
+def test_get_formula():
+    document = MockDocumentWithFormula()
+    formula = [e for e in document.elements if isinstance(e, Formula)]
+    assert isinstance(formula[0], Formula)
```

Comment:
Small nit, non-blocking, but asserting `isinstance` here doesn't seem necessary as it would have to be to be in this `formula` list. We could also just assert `formula` isn't empty.

Reply:
Sure, I already made the changes 😄